src/app/migration/src/src/app/modules/core/services/user/user.service.ts

```diff
@@ -43,12 +44,28 @@ export class UserService {
    * reference of lerner service.
    */
   public learner: LearnerService;
+  /**
+ * Reference of appId
+ */
+  public _appId: string;
+  /**
+   * Reference of channel
+   */
+  public channel: string;
```

Comment:
this should be private, get method is created to access this

Reply:
done